### PR TITLE
Prevent standard purchase receipt from being sent when resending purchase receipt from Payment History page.

### PR DESCRIPTION
**Pull request by dipakcg**
Prevent standard purchase receipt from being sent when resending purchase receipt from Payment History page.

### CODE REVIEW

**Comment by cklosowski** [comment]
@dipakcg can you please remove all changelog and version bumps? We'll decide when we do a release and version bump once we've looked over all other open issues so we can possibly include other changes.

**Comment by dipakcg** [comment]
Removed changelog and version bumps, as requested.

File: includes/email-functions.php [hunk 1]
```diff
@@ -37,6 +37,9 @@ function edd_ppe_trigger_purchase_receipt( $payment_id ) {
 function edd_ppe_resend_custom_purchase_receipts( $data ) {
 	$purchase_id = $data['purchase_id'];
 	edd_ppe_email_custom_purchase_receipts( $purchase_id, false ); // doesn't send admin email
+	
+	// Disable standard purchase receipt
```
Comment (cklosowski) [inline includes/email-functions.php]: Ok testing this out a bit, seems to be working fine, but we need to check the option in Per Product Emails for sending standard emails.

This should only disable the re-sending of the standard purchase receipt in a few cases.
1) If the option is 'unchecked' in `Settings > Extensions > Per Product Emails` for `Disable Standard Purchase Receipt` then it shouldn't be sent.
2) If there are other items in the purchase that do not have custom emails, the default one still needs to be sent.

Comment (dipakcg) [reply includes/email-functions.php]: Added conditional check when disabling standard purchase receipt from being sent.

**Comment by dipakcg** [comment]
As @cklosowski asked, added conditional check when disabling standard purchase receipt from being sent.

**Comment by cklosowski** [comment]
This looks good thanks @dipakcg I'm going to close this PR and open a new one with some whitespace fixes. Your commit history will remain intact.

**Pull request closed without merging**